openscenario/openscenario_interpreter/src/procedure.cpp

```diff
@@ -12,32 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <memory>
-#include <openscenario_interpreter/procedure.hpp>
+#include <openscenario_interpreter/simulator_core.hpp>
 
 namespace openscenario_interpreter
 {
-std::unique_ptr<traffic_simulator::API> connection = nullptr;
```

Comment:
[It](https://github.com/tier4/scenario_simulator_v2/blob/refactor/interpreter/simulator-core/openscenario/openscenario_interpreter/src/procedure.cpp) looks, this file is empty.
Why is this file remaining?